### DISCUSSION TITLE
[discussion cancelled]

### DISCUSSION THREAD

**Discussion by ghost** [RHVoice/RHVoice#852] [Q&A]
Please forget that this topic ever existed. I refuse to iterate further. Goodbye.

**Comment by zstanecic** [comment]
Hi,

It is not a genuine question. Microsoft Anna is not, and it never be a voice which can be used by this system for obvious reasons.

 

 

From: LCMCProIncUser001 ***@***.***> 
Sent: Friday, February 16, 2024 6:22 AM
To: RHVoice/RHVoice ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/RHVoice] trying to look for Microsoft Anna voice from Win 7 (Discussion #852)

 

I will not try to annoy anyone, but this is a genuine question: If I wanted to use Microsoft Anna for TTS use, is this the application to use it on?
I know this sounds a bit... strange, but I do need help figuring this out. Much appreciated.
